<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeseaWarheadMainCtrlClass</class>
 <widget class="QMainWindow" name="DeseaWarheadMainCtrlClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>战斗部建模及威力分析</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionImport"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionExport"/>
   <addaction name="actionExportPower"/>
   <addaction name="actionProperty"/>
   <addaction name="actionMetal"/>
   <addaction name="actionCharge"/>
   <addaction name="actionPower"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="deseawarheadmainctrl.qrc">
     <normaloff>:/root/new</normaloff>:/root/new</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="deseawarheadmainctrl.qrc">
     <normaloff>:/root/open</normaloff>:/root/open</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="deseawarheadmainctrl.qrc">
     <normaloff>:/root/import</normaloff>:/root/import</iconset>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
   <property name="toolTip">
    <string>导入</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="deseawarheadmainctrl.qrc">
     <normaloff>:/root/export</normaloff>:/root/export</iconset>
   </property>
   <property name="text">
    <string>导出结构</string>
   </property>
   <property name="toolTip">
    <string>导出结构</string>
   </property>
  </action>
  <action name="actionExportPower">
   <property name="icon">
    <iconset resource="deseawarheadmainctrl.qrc">
     <normaloff>:/root/export_power</normaloff>:/root/export_power</iconset>
   </property>
   <property name="text">
    <string>导出威力</string>
   </property>
   <property name="toolTip">
    <string>导出威力</string>
   </property>
  </action>
  <action name="actionMetal">
   <property name="icon">
    <iconset resource="deseawarheadmainctrl.qrc">
     <normaloff>:/root/metal</normaloff>:/root/metal</iconset>
   </property>
   <property name="text">
    <string>金属材料</string>
   </property>
   <property name="toolTip">
    <string>金属材料</string>
   </property>
  </action>
  <action name="actionCharge">
   <property name="icon">
    <iconset resource="deseawarheadmainctrl.qrc">
     <normaloff>:/root/charge</normaloff>:/root/charge</iconset>
   </property>
   <property name="text">
    <string>装药材料</string>
   </property>
   <property name="toolTip">
    <string>装药材料</string>
   </property>
  </action>
  <action name="actionProperty">
   <property name="icon">
    <iconset resource="deseawarheadmainctrl.qrc">
     <normaloff>:/root/property</normaloff>:/root/property</iconset>
   </property>
   <property name="text">
    <string>战斗部属性</string>
   </property>
   <property name="toolTip">
    <string>战斗部属性</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="deseawarheadmainctrl.qrc">
     <normaloff>:/root/save_as</normaloff>:/root/save_as</iconset>
   </property>
   <property name="text">
    <string>另存</string>
   </property>
   <property name="toolTip">
    <string>另存</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="deseawarheadmainctrl.qrc">
     <normaloff>:/root/save</normaloff>:/root/save</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="actionPower">
   <property name="text">
    <string>威力分析</string>
   </property>
   <property name="toolTip">
    <string>威力分析</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="deseawarheadmainctrl.qrc"/>
 </resources>
 <connections/>
</ui>
